src/sass/components/_header.scss

```diff
@@ -167,122 +168,227 @@ header {
         }
       }
     }
+    
+    .search-wrapper {
+      display: flex;
+      justify-content: center;
+      align-items: center;
 
-    .search-form {
-      border: 1px solid $form-border-color;
-      display: inline-flex;
-      height: 45px;
-      color: $text-color;
+      .category-list-show {
+        display: none;
+      }
 
-      .category {
-        border-right: 1px solid $form-border-color;
-        display: flex;
-        align-items: center;
-        position: relative;
-        padding: 0 30px 0 35px;
+      .search-form {
+        border: 1px solid $form-border-color;
+        display: inline-flex;
+        height: 45px;
+        color: $text-color;
 
-        i:first-child {
-          color: $primary;
-          position: absolute;
-          left: 10px;
-          z-index: 0;
+        %category-hover {
```

Comment:
Tutaj na pewno ma być `%` ?

Reply:
Użyłem % ponieważ .category (oraz i wewnątrz) i dziecko .category-element mają taki sam :hover, a nie chciałem go powtarzać. Czy powinienem napisać to w inny sposób? 
Chyba, że chodzi o miejsce % w kodzie. Jeśli tak to przeniosłem na początek pliku.